gcloud-java-storage/src/test/java/com/google/gcloud/storage/ITStorageTest.java

```diff
@@ -317,16 +317,14 @@ public void testComposeBlobFail() {
   @Test
   public void testCopyBlob() {
     String sourceBlobName = "test-copy-blob-source";
-    BlobInfo blob = BlobInfo.builder(BUCKET, sourceBlobName).build();
-    assertNotNull(storage.create(blob, BLOB_BYTE_CONTENT));
+    BlobId source = BlobId.of(BUCKET, sourceBlobName);
+    assertNotNull(storage.create(BlobInfo.builder(source).build(), BLOB_BYTE_CONTENT));
     String targetBlobName = "test-copy-blob-target";
-    Storage.CopyRequest req = Storage.CopyRequest.of(blob.blobId(), targetBlobName);
-    BlobInfo remoteBlob = storage.copy(req);
-    assertNotNull(remoteBlob);
-    assertEquals(BUCKET, remoteBlob.bucket());
-    assertEquals(targetBlobName, remoteBlob.name());
-    byte[] readBytes = storage.readAllBytes(BUCKET, targetBlobName);
-    assertArrayEquals(BLOB_BYTE_CONTENT, readBytes);
+    BlobInfo target = BlobInfo.builder(BUCKET, targetBlobName).contentType(CONTENT_TYPE).build();
```

Comment:
Maybe in this one we should not override the content-type (and keep the next one to test the override)?